samples/matter/light_switch/src/light_switch.cpp

```diff
@@ -53,7 +53,7 @@ void LightSwitch::InitiateActionSwitch(Action action)
 			Platform::Delete(data);
 			return;
 		}
-		data->IsGroup = Nrf::Matter::BindingHandler::IsGroupBound();
+		data->IsGroup.SetValue(Nrf::Matter::BindingHandler::IsGroupBound());
```

Comment:
I also had impression this IsGroup parameter should be removed at all. Nevertheless I think it was added in the past for the testing purposes, so we would be able to distinguish whether we want to apply only command to a group or only to a unicast if we have both types of entries in the binding table. If QA guys do not need it I would be ok with removing it at all.

Reply:
> It's like telling a binding table what's inside the binding table even though the binding table has enough information to process the request.

Not quite, it does sort of filterting, so multicast binding entries will not be handled if IsGroup is not set, which reflects sort of user's request. And in the opposite way for the unicasts, it won't be send even if we have such binding entry, but IsGroup is set.

Reply:
Yeah, but in this case there's no user request - there's a single button. So what it only does is that if there are both unicast and multicast bindings - only multicast will be processed (which I'm not sure is desired). I agree that for shell commands it might be useful to force the binding type (not sure if any tests make use of that though).

Reply:
Ah right, in this case it should be removed. I thought you mean the overall idea of using this flag (including commands).

Reply:
I removed it and also the `IsGroupBound` method that imo was not useful.

Reply:
Thanks!